### ISSUE TITLE
Retrieve paired-end reads with their mates

### ISSUE THREAD

**Issue by iquasere** [lh3/seqtk#156]
Greetings!

I wanted to know if, when retrieving paired reads with seqtk from a list of names,there is a way to retrieve the paired mate as well.

Thank you for your attention!

**Comment by schultzm** [comment]
Yes.  Run it twice.  First time, use R1, second time use R2.  On each run, use exactly the same seed value.  This use case is given as an example in the user manual.
